iina/de.lproj/AboutWindowController.strings

```diff
@@ -1,27 +1,12 @@
 
-/* Class = "NSTextFieldCell"; title = "Label"; ObjectID = "5xy-6V-bAd"; */
-"5xy-6V-bAd.title" = "Label";
-
-/* Class = "NSTextFieldCell"; title = "Website:"; ObjectID = "AfN-ft-1xe"; */
-"AfN-ft-1xe.title" = "Webseite:";
-
 /* Class = "NSWindow"; title = "About"; ObjectID = "F0z-JX-Cv5"; */
-"F0z-JX-Cv5.title" = "Über";
-
-/* Class = "NSTextFieldCell"; title = "Label"; ObjectID = "L8a-Ak-JW7"; */
-"L8a-Ak-JW7.title" = "Label";
+"F0z-JX-Cv5.title" = "Über IINA";
 
-/* Class = "NSTextFieldCell"; title = "Label"; ObjectID = "Nd4-sj-z8a"; */
-"Nd4-sj-z8a.title" = "Label";
+/* Class = "NSTextFieldCell"; title = "This is an alpha release and not recommended for daily use. Please check for updates frequently. For bug reports, feature requests or help, please create an issue on Github or email me."; ObjectID = "TI7-3V-nUn"; */
+"TI7-3V-nUn.title" = "Dies ist Ein Programm im Aplpha Stadium und von daher noch nich für den täglichen Gebrauch gedacht. Bitte regelmäßig nach Aktualisierungen suchen.  Um Fehler zu melden oder für Vorschläge zur Verbessung des Programms bitte auf Github einen Fall erzeugen oder mir eine E-Mail schreiben.";
```

Comment:
1. Typo: "Ein" must be "ein".
2. Typo: "nich" must be "nicht". 
3. Extra whitespace between the 2 sentences.